### ISSUE TITLE
Feature request: email confirmation  resent.

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/templating#218]
We have lots of customer complaints at the end [Account confirmation and password recovery in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/accconfirm?tabs=aspnetcore2x%2Csql-server)

>  I found it odd that, when requiring email confirmation, if for whatever reason the user misses the original email with the confirmation link, the login view reminds you but there is no way to get the app to resend the confirmation.  

> I understand in some cases users miss to confirm their email account, and they should be able to see the option of "Resend Email Confirmation Link" when they try to login. Unfortunately I could not find any documentation to solve this problem, but what I did. I created a controller with Read, Delete action methods that can be used to delete unconfirmed email accounts from admin panel of my application. And then any unverified email can be used again to register in our application. I think that there is a better way to send confirmation link again through the application which I don't know.

**Comment by Rick-Anderson** [comment]
How is this for a workaround?

Model a solution by looking at the following methods on the Account Controller, `ForgotPassword`, and `Register`.  The register method has the code generating the token (code) and callback for sending the email confirmation.  Create get, post methods named `ResendEmailConfirmation` on the Accounts controller.
@HaoK

**Comment by egmfrs** [comment]
There is also a `SendVerificationEmail` POST action in the Manage controller

**Comment by javiercn** [comment]
@mkArtakMSFT This is more suitable for the Identity repo. @blowdart should chip-in before we do anything.

**Comment by blowdart** [comment]
This would need careful planning, to throttle emails so as not to spam someone, as someone could just keep hitting "I've lost my password", so this is feature work rather than a bug fix.

**Comment by mkArtakMSFT** [comment]
Good point. That makes me believe this doesn't meet the priority bar for 2.2 release. Should we consider this for 3.0?

**Comment by Rick-Anderson** [comment]
Flood of "lost password" email is not as problematic as DOS account lockout attack (see https://github.com/aspnet/Identity/pull/1914) which we currently have.

For those willing to risk *Flood of lost password emails* can I get a work around to document until this feature is supported?

> How is this for a workaround?

> Model a solution by looking at the following methods, `ForgotPassword`, and `Register`.  The register method has the code generating the token (code) and callback for sending the email confirmation.  Create get, post methods named `ResendEmailConfirmation`.
